README.md

```diff
@@ -1,25 +1,34 @@
-# Getting Started with Create React App
+# Covid Tracker
```

Comment:
- [ ] You have not used [the correct gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) as provided in the project requirements. Your Pull Request should be from **feature** branch to **dev** branch. Kindly fix this.